substation-diagram/substation-diagram-core/src/main/java/com/powsybl/substationdiagram/svg/GraphMetadata.java

```diff
@@ -34,6 +35,8 @@
 
         private ComponentType componentType;
 
+        private Double rotationAngle;
```

Comment:
rotated boolean is not needed anymore?

Reply:
It is still used, in combination with rotationAngle, to calculate the right anchorPoints in WireConnection.java